.github/workflows/eks-registry-2.1.yml

```diff
@@ -0,0 +1,169 @@
+name: "[WIP|PC|Weekly]SUSE Private Registry:2.1"
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - aks
+
+env:
+  SPR_VERSION: "2.1"
+  CHART_SRC: "registry.suse.com/harbor/harbor:1.5"
+
+jobs:
+  public_cloud_registry:
+    name: Integration Tests
+    runs-on: ecosystem-ci-runners
+    container:
+      image: "opensuse/leap:15.2"
+      # ensures a dedicated empty workspace
+      volumes:
+        - /__w/caps-automation
+    strategy:
+      fail-fast: false
+      matrix:
+        k8s_distribution: [aks]
+        test_scenario: ["ha+tls+external-db_redis"]
+        include:
+          - k8s_distribution: aks
+            k8s_storageclass: azurefile
+    timeout-minutes: 90
+    steps:
+      - name: Setup environment
+        id: set_env
+        run: |
+          echo "NAMESPACE=spr${SPR_VERSION/./}-release-${{ github.run_number }}" >> $GITHUB_ENV
+          echo "::set-output name=chart_install_src::${CHART_SRC}"
+
+      - name: Install requirements
+        run: |
+          zypper -n ar -f http://download.suse.de/ibs/SUSE:/CA/openSUSE_Leap_15.2/SUSE:CA.repo
+          zypper -n ar -Gf http://download.suse.de/ibs/SUSE:/SLE-15-SP2:/Update:/Products:/CAPS-Registry:/${SPR_VERSION}/standard/SUSE:SLE-15-SP2:Update:Products:CAPS-Registry:${SPR_VERSION}.repo
+          zypper -n in ca-certificates-suse gettext gzip helm jq kubernetes-client \
+            python3-robotframework tar unzip wget
+          wget -q https://releases.hashicorp.com/terraform/0.13.4/terraform_0.13.4_linux_amd64.zip \
+            -O terraform.zip
+          unzip terraform.zip -d /usr/local/bin
+
+      - uses: actions/checkout@v2
+
+      - name: Setup ${{ matrix.k8s_distribution }} for ${{ matrix.test_scenario }}
+        env:
+          ARM_CLIENT_ID: ${{ secrets.TF_AZ_ARM_CLIENT_ID }}
+          ARM_CLIENT_SECRET: ${{ secrets.TF_AZ_ARM_CLIENT_SECRET }}
+          ARM_SUBSCRIPTION_ID: ${{ secrets.TF_AZ_ARM_SUBSCRIPTION_ID }}
+          ARM_TENANT_ID: ${{ secrets.TF_AZ_ARM_TENANT_ID }}
+        run: |
+          terraform init -backend-config="storage_account_name=ecosystemci" \
+            -backend-config="container_name=tfstate" \
+            -backend-config="access_key=${{ secrets.TF_AZ_BACKEND_ACCESS_KEY }}" \
+            -backend-config="key=codelab.microsoft.tfstate"
+          terraform workspace new ${NAMESPACE}
+          terraform apply -var-file ../../../tests/registry/${{ matrix.test_scenario }}/infra.tfvars \
+            -auto-approve
+          echo "KUBECONFIG=$(realpath ./azurek8s)" >> $GITHUB_ENV
```

Comment:
I assume the config is not called `azurek8s`

Reply:
fixed